lesson08/assignment/assignment08/src/components/AboutPage.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+
+  <div class="about">
+    <body>
+      <main id="main-content" class="row justify-content-center mb-5">
+        <section class="about-text">
+          <h2 class="text-uppercase text-center">About Us</h2>
+          <p class="mb-5">With exceptional espresso, pastries baked from scratch, gluten-free options, house-made granola, and delicious panini, Ten Ton Art & Coffee has what you crave. A perfect choice for breakfast, lunch, or a quick pick-me-up.</p>
+          <p>Coffee shops in towns and villages throughout the world always seem to be community gathering places, and Ten Ton is no exception. Join us!</p>
+        </section>
+        <!-- INSTAGRAM COMPONENT -->
+        <InstagramFeed :images=images />
```

Comment:
What's here works fine, but it would be preferable for the iteration through `images` to happen here, and the component to receive an image object, so url, alt, title, etc props could be validated.